rdtools/analysis_chains.py

```diff
@@ -373,7 +373,9 @@ def _filter(self, energy_normalized, case):
         -------
         None
         '''
-        bool_filter = True
+        filter_components = pd.DataFrame({
+            'default': pd.Series(True, index=energy_normalized.index)
+        })
```

Comment:
Do we need this default column? Could we just do:
`filter_components = pd.DataFrame(index=energy_normalized.index)`
This would save us dropping the default column below. On the other hand, what about a case where someone turns off all the filtering? In that case maybe we want the `default` column and should retain it below.

Reply:
Yeah I think I was anticipating the case where all filtering is disabled, although now that I'm looking at it I don't think it's actually needed in the degenerate no-filter case?  `pd.DataFrame(index=[1,2,3]).all(axis=1)` returns all `True` for me on pandas `0.25.1` and `1.0.3`.  I'll change to your suggestion.